### ISSUE TITLE
Explanation of fine tuning

### ISSUE THREAD

**Issue by mixmixmix** [ctorney/uavTracker#7]
Fine tuning is defined as following in the original code (before any recent changes):
https://github.com/ctorney/uavTracker/blob/f44941d9fd5cc37e63e030a303a789ac2c55152d/train/train.py#L32
```
if FINE_TUNE:
    BATCH_SIZE= 4
    EPOCHS=100
    model = get_yolo_model(IMAGE_W,IMAGE_H, num_class=1,headtrainable=True, trainable=True)
    model.load_weights('../weights/horses-yolo.h5')
else:
    BATCH_SIZE=32
    EPOCHS=500
    model = get_yolo_model(IMAGE_W,IMAGE_H, num_class=1,headtrainable=True)
model.load_weights('../weights/yolo-v3-coco.h5', by_name=True)
```

@ctorney, could you clarify:
1. When we _fine tune_ we are training all layers. 
2. The _general_ training seems to take the generic COCO weights but it only re-trains layers above 80. 

I would think it should be other way round. (?)

**Comment by ctorney** [comment]
That's right - step 1 is general training. All lower layers are frozen and only the final dense layers are trained. Fine tuning is when the whole network is unfrozen and the learning rate is small. The sequence of training should be kept - train head layers first with lower layers frozen, then train all layers with a small learn rate - but whether fine tuning is a good name for the second stage is debatable

**Comment by mixmixmix** [comment]
Ok, I understand now. Both of those steps are types of fine tuning, so we can just call them "phase 1 fine tuning" (strong tune, top layers) and "phase 2 fine tuning" (weak tune, all layers)

**Comment by mixmixmix** [comment]
compare with: Building powerful image classification models using very little data
https://blog.keras.io/building-powerful-image-classification-models-using-very-little-data.html

**Comment by mixmixmix** [comment]
So it seems that our approach is unorthodox but works well (maybe overfitting?) 
To be experimentally proven.
